src/controllers/update/update.manage.trusts.orchestrator.controller.ts

```diff
@@ -60,6 +61,38 @@ const shouldGoToReviewTheTrust = (trustInReview: Trust) => {
   return !trustInReview.review_status?.reviewed_trust_details;
 };
 
+export const trustChangeHandler = async (req: Request, res: Response, next: NextFunction) => {
+  logger.debugRequest(req, `${req.method} ${req.route.path}`);
+
+  try {
+    const trustId = req.params[ROUTE_PARAM_TRUST_ID];
+    const appData: ApplicationData = getApplicationData(req.session);
+
+    await putTrustInChangeScenario(req, appData, trustId);
+
+    res.redirect(UPDATE_MANAGE_TRUSTS_REVIEW_THE_TRUST_URL);
+  } catch (error) {
+    next(error);
+    return;
+  }
+};
+
+const putTrustInChangeScenario = async (req: Request, appData: ApplicationData, trustId: string) => {
+  const trustForChangeScenario = appData.trusts?.splice(appData.trusts.findIndex(trust => trust.trust_id === trustId), 1)[0];
+  appData.update?.review_trusts?.push(trustForChangeScenario as Trust);
+  const trust = getTrustInReview(appData) as Trust;
+
+  trust.review_status = {
```

Comment:
Instead of setting all to true possibly just change the last one or set trust.review_status == undefined?

Reply:
and could even move the trust over to the main list. Then 1064 would not need to do the findIndex/splice.

Reply:
It's been "reviewed" so just changes from now on. Fine with keeping it "in review" but seems strange to have all the flags set to true already.

Reply:
I have to set the others to true as well, otherwise the user after submitting will be forced to go through all the other pages as well and re-review the trustees etc. 
So by setting others to true and only reviewed_trust_details to false, the user's flow will be as required after clicking a change link - "review the trust" -> "individuals and entities involved"

Reply:
I'm somewhat concerned that the user could skip through the review by going through this and having all (but one of) the flags set to true. If we are sure that won't happen then ok.